subsys/suit/Kconfig

```diff
@@ -6,7 +6,6 @@
 
 menuconfig SUIT
 	bool "Enable SUIT modules"
-	depends on !SOC_NRF54H20_IRON
```

Comment:
Wouldn't it be time to delete the suit path in here as well then? or at least blacklist us for the `SOC_NRF54H20`?